src/span_context.js

```diff
@@ -11,59 +11,6 @@
  * <trace_id, span_id, sampled> tuple).
  */
 export default class SpanContext {
```

Comment:
For what it's worth, I'm fine with `SpanContext` having no methods. I'd like to do something this week to make the implementation code more inheritance-based than purely implicit per #35 and I think the "empty" class will fit in better then (and if it *doesn't* I can remove it then).